packages/listener-solidarity/src/components/User/composeUsers.ts

```diff
@@ -143,6 +143,8 @@ export default async (
         widget.id,
         instance["accept_terms"]
       );
+    } else if (instance["condition"]) {
+      register["user_fields"]["condition"] = instance["condition"];  
     } else {
       register["user_fields"]["condition"] = "cadastrada";
     }
```

Comment:
podemos so criar outro if, pode tirar o else

ficaria tipo 

```suggestion
    } else if (instance["condition"]) { // maps volunteer condition from new volunteer register form
      register["user_fields"]["condition"] = instance["condition"];  
    } 
```